modules/mesh_com/entrypoint.sh

```diff
@@ -1,9 +1,22 @@
 #!/bin/bash
```

Comment:
Should this be ```bash
#!/bin/bash -eu
```

It has the same effect as `set -eu`

Where the `e` means that the script will stop if any command fails. It is probably not intended behaviour that if `$ mesh-ibss.sh ap` fails we still continue to starting `mesh_executor`

Reply:
You are right. That is needed to make failing visible

Reply:
Left unbound variable check out